x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -177,8 +179,8 @@ public final void test() throws Throwable {
     }
 
     protected void shouldSkipTest(String testName) throws IOException {
-        if (testCase.requiredCapabilities.contains("semantic_text_field_caps") || testCase.requiredCapabilities.contains("rerank")) {
-            assumeTrue("Inference test service needs to be supported for semantic_text", supportsInferenceTestService());
+        if (Stream.of("semantic_text_field_caps", RERANK.capabilityName()).anyMatch(testCase.requiredCapabilities::contains)) {
```

Comment:
nit: we should make this consistent and use `SEMANITC_TEXT_FIELD_CAPS.capabilityName()` if we use `RERANK.capabilityName()`.